server/store/store.go

```diff
@@ -139,9 +139,9 @@ type Store interface {
 	StepByUUID(string) (*model.Step, error)
 	StepChild(*model.Pipeline, int, string) (*model.Step, error)
 	StepList(*model.Pipeline) ([]*model.Step, error)
-	StepCreate([]*model.Step) error
 	StepUpdate(*model.Step) error
 	StepClear(*model.Pipeline) error
+	StepListFromWorkflow(*model.Workflow) ([]*model.Step, error)
```

Comment:
Could be helpful to call it `StepListFromWorkflowFind` or `StepListFromWorkflowGet` etc to know that its not setting, but getting data.